crates/voicevox_core/src/user_dict/part_of_speech_data.rs

```diff
@@ -0,0 +1,121 @@
+use derive_getters::Getters;
+use once_cell::sync::Lazy;
+use std::collections::HashMap;
+
+use crate::UserDictWordType;
+
+/// 最小の優先度
+pub static MIN_PRIORITY: u32 = 0;
+/// 最大の優先度
+pub static MAX_PRIORITY: u32 = 10;
+
+/// 品詞ごとの情報
+#[derive(Debug, Getters)]
+pub struct PartOfSpeechDetail {
+    /// 品詞
+    pub part_of_speech: &'static str,
+    /// 品詞細分類1
+    pub part_of_speech_detail_1: &'static str,
+    /// 品詞細分類2
+    pub part_of_speech_detail_2: &'static str,
+    /// 品詞細分類3
+    pub part_of_speech_detail_3: &'static str,
+    /// 文脈IDは辞書の左・右文脈IDのこと
+    /// 参考: https://github.com/VOICEVOX/open_jtalk/blob/427cfd761b78efb6094bea3c5bb8c968f0d711ab/src/mecab-naist-jdic/_left-id.def
```

Comment:
```suggestion
    /// 文脈IDは辞書の左・右文脈IDのこと
    ///
    /// 参考: <https://github.com/VOICEVOX/open_jtalk/blob/427cfd761b78efb6094bea3c5bb8c968f0d711ab/src/mecab-naist-jdic/_left-id.def>
```

![image](https://github.com/VOICEVOX/voicevox_core/assets/14125495/4ca8ba64-003c-4732-b4fd-dda6b380c440)